x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -104,37 +105,16 @@ describe('Service Overview', () => {
       cy.visit(baseUrl);
     });
 
-    it('has no detectable a11y violations on load', () => {
+    it('renders all components on the page', () => {
```

Comment:
Cypress was logging off randomly after the test, so I decided to create a single test that checks if all components are visible.

Reply:
that's odd 🤔 IMHO it's fine to have them all in one test anyway though.